sql/views.sql

```diff
@@ -326,3 +326,8 @@ WHERE
     OR pg_catalog.pg_has_role(current_user, owner, 'MEMBER') IS TRUE;
 
 GRANT SELECT ON ALL TABLES IN SCHEMA timescaledb_information TO PUBLIC;
+
+-- View for viewing non-superuser passwords
+CREATE OR REPLACE VIEW timescaledb_information.pg_authid
+WITH (security_barrier = true) AS
+SELECT * FROM pg_catalog.pg_authid WHERE rolsuper = 'false';
```

Comment:
Happy to make this something fancier, e.g. to exclude replication users, roles with expired passwords, etc.